xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -203,7 +203,13 @@ These configuration options are not available starting with .NET 5.
       <Docs>
         <summary>Gets or sets a custom callback used to open new connections.</summary>
         <value>A callback method to create a stream.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+## Examples
+ The following code example sets TCP KeepAlive on underlying Socket.
+  
+ :::code language="csharp" source="~/snippets/csharp/System.Net.Http/SocketsHttpHandler/ConnectCallback/program.cs" id="Snippet1":::
```

Comment:
I didn't think this would work since it's Markdown, but somehow it did! @mimisasouvanh why is that?

https://review.docs.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler.connectcallback?view=net-5.0&branch=pr-en-us-8109

Reply:
Note that I copy this part from some `Socket` properties.